pennylane/qnode_new/base.py

```diff
@@ -83,7 +83,7 @@ def _get_signature(func):
     func.n_pos = n_pos
 
 
-class QNode:
+class BaseQNode:
```

Comment:
Maybe it would have been nice to separate the renaming part into a different PR, since then it could have been easy to follow up on when this change was carried out.

Could maybe the name of the PR still be modified accordingly?